pages/sign-up-page.pug

```diff
@@ -0,0 +1,33 @@
+extend ../pages/layout
+block page
+	include ../pages/partials/header-info
+block unique content
+	+showNav('Trang chủ / ', '', 'Đăng nhập')
+	div.sign-up
+		+tittle('ĐĂNG KÝ', 'assets/image/titleleft-dark.png')
+		+button('ĐĂNG KÝ')
+		form(action="/action_page.php")
+			h3 Thông tin cá nhân
+			div.form
+				div.form-label
+					label(for='fname') Tên trước *
+					label(for='lname') Tên sau *
+					label(for='email') Email *
+					label(for='password') Password *
+				div.form-input
+					input(type='text' name="fname" required)
```

Comment:
Thuộc tính for của label tương ứng với id của input chị nhé
```suggestion
label(for="fname") Tên trước *
input#fname(type='text' name="fname" required)
```

Reply:
okie em